feature.h

```diff
@@ -190,6 +190,126 @@ S_enable_feature_bundle(pTHX_ SV *ver)
 }
 #endif /* PERL_IN_OP_C */
 
+#ifdef PERL_IN_MG_C
+
+#define memEQ_prefix(key, len, s) \
+  ((len) >= sizeof("" s)-1 && memcmp((key), "" s, sizeof("" s)-1) == 0)
```

Comment:
This looks a lot like the memBEGIN() macro in handy.h

Reply:
thanks, fixed.